apps/web/frontend/components/projectForm/projectForm.tsx

```diff
@@ -55,15 +56,26 @@ const projectInputSchema: z.ZodSchema<ProjectInput> = projectInputValidations
     }
   })
 
+export const ProjectFormFragment = graphql(`
+  fragment ProjectForm on Project {
+    title
+    startDate
+    endDate
+    canModify
+    ...DeleteProjectModal
```

Comment:
we have to take good care about naming. Naming the fragments like the components confused me for a moment but I think now it is ok because it is something different inside gql and inside ts code